doc/experiment.rst

```diff
@@ -12,316 +38,211 @@ Jupyter notebook, interactive python session, or Python file and use the
+    exp = Experiment("name-of-experiment", launcher="cobalt") # Cobalt launcher
+    exp = Experiment("name-of-experiment", launcher="lsf")    # LSF launcher
+    exp = Experiment("name-of-experiment", launcher="auto")   # auto-detect launcher
+
+
+=========
+ Entities
+=========
+
+Defining workflow stages requires the utilization of functions
+associated with the ``Experiment`` object. The ``Experiment`` object
+is intended to be instantiated once and utilized throughout
+the workflow runtime. In the following content, we define five
+entities created by an experiment: ``Orchestrator``, ``Model``, ``Ensemble``,
+``RunSettings``, and ``BatchSettings``, along with a discussion of their relationships.
+
+Entity Relationship Diagram
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Below is an Entity Relationship Diagram using Crow's Foot Notation.
+Moving from the top to the bottom of the diagram, you will first notice that
+an Experiment can have none or many ``Orchestrators``, ``Models``, and ``Ensembles``.
+The Experiment class is responsible for all entity initialization functions:
+
+1. ``Experiment.create_orchestrator()`` :raw-html:`&rarr; ``Orchestrator``
+2. ``Experiment.create_model()`` :raw-html:`&rarr; ``Model``
+3. ``Experiment.create_ensemble()`` :raw-html:`&rarr; ``Ensemble``
+4. ``Experiment.create_run_settings()`` :raw-html:`&rarr; ``RunSettings``
+5. ``Experiment.create_batch_settings()`` :raw-html:`&rarr; ``BatchSettings``
+
+.. |SmartSim ERD| image:: images/edr.png
+  :width: 700
+  :alt: Alternative text
 
-    #!/bin/bash
-    #SBATCH --exclusive
-    #SBATCH --nodes=1
-    #SBATCH --ntasks-per-node=32
-    #SBATCH --time=00:10:00
+|SmartSim ERD|
+
+Moving on to the ``Model`` entity class, you will notice that a ``Model`` requires a
+minimum and maximum of one ``RunSetting`` object. However, it accepts a minimum of zero
+and a maximum of one ``BatchSetting`` object.
+
+The ``Ensemble`` entity class does not require either ``RunSetting`` or ``BatchSetting``
+objects upon initialization but does accept a maximum of one for each.
+
+Orchestrator
+^^^^^^^^^^^^
+The ``Orchestrator`` is an in-memory database that can be launched alongside
+``Model`` and ``Ensemble`` entities in SmartSim. The ``Orchestrator`` can be used to store and retrieve
+data during the course of an experiment. In order to stream data into or
+receive data from the Orchestrator, one of the SmartSim clients
+(SmartRedis) has to be used within a Model.
+
+To create a standard orchestrator, use the ``Experiment.create_database()``
+function. A standard orchestrator will run the database and applications on
+separate nodes. The database can be single or multi-sharded.
+For a colocated orchestrator, use the ``model.colocated_db()``
+function provided by the Model object. This action results in
+the launch of an orchestrator on the same node as the application
+but is limited to single-sharded database configuration.
```

Comment:
I think having nice graphics for the different deployment options for the Orchestrator would be helpful for users.  Let's adapt some existing graphics we might have.